contrib/cirrus/runner.sh

```diff
@@ -398,9 +398,7 @@ dotest() {
 }
 
 _run_machine() {
-    # TODO: This is a manually-triggered task, if that ever changes need to
-    # add something like:
-    # _bail_if_test_can_be_skipped docs test/e2e test/system test/python
+    _bail_if_test_can_be_skipped docs test/apiv2
```

Comment:
The correct incantation here is `_bail... pkg/machine/e2e` (no `docs`, no `test/apiv2`). And the commit message should be changed accordingly. And of course this can't merge until #15031 is fixed.